### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by minj** [minj/foxtrick#1109]
Original [issue 1109](https://code.google.com/p/foxtrick/issues/detail?id=1109) created by [geozerogm](mailto:geozerogm@gmail.com) on 2013-06-13T22:34:06.000Z:

Hi! It's not an issue really as a request. There are some users who are visual impaired. This doesn't seems to bother them and they enjoy the game as everyone. But talking with a very good friend of mine he told me that he can't use Foxtrick. He uses two programms to read, the Jaws 13.0 and the NVDA. With both Foxtrick is not accessible. I believe it will be nice if it is helpful for those people also as for al lot of users. Thanks for your time GM-geozero.

---
ZenHub Epic for accessibility improvements

**Comment by bkoray** [comment]
I'd like to revive this issue to discuss possible improvements and bring it to your attention. I am a visually impaired user using JAWS 2019, and unfortunately this extention makes some pages completely unusable see: players page and individual page of a player. I am thinking that lightweight version might help but apparently it's no longer being developed see:
#1609  

With any testing or ideas, I'd be more than willing to help. Needless to say being able to use this extention would be extremely useful.

**Comment by minj** [comment]
@bkoray I am no longer sure I understood your comment correctly. Is FT still causing major issues for you or were you able to solve it on your end? Should I close other issue?

**Comment by minj** [comment]
The accessibility problem on most pages has been reported fixed. Any further problems warrant an issue of their own
